### DISCUSSION TITLE
Is there a way to add buttons to custom activity?

### DISCUSSION THREAD

**Discussion by asxlvm** [Rapptz/discord.py#6775] [Q&A]
I saw that people have custom activities with buttons
![IMG_20210422_164911](https://user-images.githubusercontent.com/81298611/115735369-b86e9000-a38a-11eb-8d5f-dc64f145ed7d.jpg)
Could I do the same for my Discord bot?

**Comment by AbstractUmbra** [comment]
This is not possible with a discord bot, no.

**Reply by asxlvm** [reply]
How so?

**Reply by asxlvm** [reply]
There are RPC clients for Discord that can do this

**Reply by AbstractUmbra** [reply]
This is not possible with a Discord bot.

This is possible with RPC on a user, but this does not exist for bots. More info can be found [here](https://discord.com/developers/docs/topics/rpc).

**Comment by Rapptz** [comment]
This functionality is not available for bots.
